packages/commonwealth/server/routes/topics/getTopics.ts

```diff
@@ -27,18 +25,36 @@ export const getTopics = async (
 
   const { community_id, count_only } = req.query;
 
-  const where = { chain_id: community_id };
+  const where = { chain_id: community_id, };
+
+  // Join in last_commented_on to thread that is most recently commented on
+  const include = [{
+    model: models.Thread,
+    as: 'threads',
+    where: {
+      [Op.and]: [Sequelize.literal(` "threads"."last_commented_on" = (
+      Select MAX(last_commented_on)
```

Comment:
Do we need latest_activity if we have last_commented_on?

Reply:
It just renames it to latest_activity, because last_commented_on is a little misleading, since there could be no comments on the thread (should we rename the field in this case)

Reply:
Then we should rename it in the model as well, right?

Reply:
Well I potentially fixed it, but it may not be at the scope you were asking for. If I had done this part wrong let me know.